Sources/SwiftLexicalLookup/SwiftLexicalLookup.docc/LookupRules.md

```diff
@@ -0,0 +1,78 @@
+# Unqualified Lookup Rules
+
+## Overview
+Unqualified lookup can be initiated from any source position, optionally targeting a specific identifier for name comparison. Unqualified lookup traverses the syntax tree from bottom-up, collecting names that match the lookup criteria. A name matches the lookup if the specified identifier (if present) refers to the introduced name and the position from which the unqualified lookup was triggered is preceded by the position where the name was introduced (if specified).
```

Comment:
> and the position from which the unqualified lookup was triggered is preceded by the position where the name was introduced (if specified)

This isn’t technically true because types can be defined after the lookup position, right?

Reply:
I tried to indicate it by having `(if specified)` by the end of the sentence, but I agree it was a bit ambiguous. I revised the ending.

Reply:
Sounds good 👍🏽